spec/extensions/forbid_lazy_load_spec.rb

```diff
@@ -1,10 +1,11 @@
 require_relative "spec_helper"
 
 describe "forbid_lazy_load plugin" do
-  before do
+  # opts are forwarded to `plugin :forbid_lazy_load`
+  def setup_model(opts={})
     @c = Class.new(Sequel::Model)
     @c.set_dataset Sequel::Model.db[:ts].with_fetch({:id=>2, :t_id=>3})
-    @c.plugin :forbid_lazy_load
+    @c.plugin :forbid_lazy_load, **opts
```

Comment:
You should pass `opts` directly here (Sequel still supports Ruby 1.9).

Reply:
Got it! Fixed.